17-refrigerator-monitor/README.md

```diff
@@ -0,0 +1,135 @@
+# Refrigerator Monitor
+
```

Comment:
To be consistent with our other app readme's, I would add a tagline that summarizes the intent.  Something like:

> Monitor refrigerator temperature, humidity, and power using a Notecard and a BME280 sensor.